retrofit/src/main/java/retrofit2/Response.java

```diff
@@ -121,8 +121,9 @@ public static <T> Response<T> success(@Nullable T body, Headers headers) {
   }
 
   private final  okhttp3.Response rawResponse;
-  private final  @Nullable @Owning T body;  //
-  private final  @Nullable @Owning ResponseBody errorBody;  //The class does not have a close method but Responsebody does have one. No way to tell checker the close() method exists outside the class. A method named "close" should be added as a more direct way to close the resource.
+  private final  @Nullable @Owning T body;
+  @SuppressWarnings("calledmethods:required.method.not.called") //Resource leak checker can not guarentee that the call errorBody() will call close. This is a flase positive because the implementation of close is held by the type of errorbody and not the class it belongs to. 
+  private final  @Nullable @Owning ResponseBody errorBody; 
```

Comment:
Where is `errorBody.close()` actually called? If not in `Response`'s close method, then where?